application/src/main/resources/application.yml

```diff
@@ -77,8 +77,8 @@ app:
     gmcUsername: ${GMC_USER_NAME:guest}
     gmcPassword: ${GMC_PASSWORD:guest}
     designatedBodies: ${DESIGNATED_BODY_CODE:1-AIIDHJ,1-AIIDMQ,1-AIIDNQ,1-AIIDMY,1-AIIDQQ,1-AIIDWT,1-AIIDR8,1-AIIDSA,1-AIIDH1,1-AIIDWA,1-AIIDVS,1-AIIDWI,1-AIIDSI}
-    recommendationstatuscheck.cronExpression: ${REC_STATUS_CHECK_CRON:0 0 * * * *}
-    nightlySyncStart.cronExpression: ${REC_NIGHTLY_SYNC_CRON:0 0 * * * *}
+    recommendationstatuscheck.cronExpression: ${REC_STATUS_CHECK_CRON:0 0 8,12,16 * * *}
```

Comment:
Ah, was just about to say we might want this to be whatever is most useful for local dev.  All good though :)